src/api-routes/persons/{osuId}/addresses.js

```diff
@@ -12,11 +12,12 @@ const get = async (req, res) => {
   try {
     const { query, params: { osuId } } = req;
 
-    if (!await personExists(osuId)) {
+    const pidm = await personExists(osuId);
```

Comment:
We probably shouldn't use `pidm` as variable name since it's also Ellucian defined column name. I'd recommend changing it to  something like `personId` or `internalId`.

Reply:
I'll make the change but not sure why it being Ellucian defined is a problem. Is it supposed to be private info or something?

Reply:
Yeah, based on the contract, we shouldn't expose table name and column name outside. Just using pidm should probably be fine since it's not the full column name, but I guess naming it implicitly will be safer.